components/filters/KeywordFilter.vue

```diff
@@ -17,30 +16,29 @@
           placeholder="Add keyword and press Enter"
           class="w-full px-3 py-1 border-0 bg-transparent text-slate-900 dark:text-white focus:outline-none focus:ring-0" />
       </div>
-    </div>
-    <p class="mt-1 text-xs text-slate-500 dark:text-slate-400">{{ $t('addKey') }}</p>
-  </div>
+    </UiDefaultInputBox>
 
+    <p class="mt-1 text-xs text-slate-500 dark:text-slate-400">{{ footer }}</p>
+  </div>
 </template>
 
 <script setup lang='ts'>
-const props = defineProps({
-  filter: {
-    type: Object,
-    required: true
-  }
-})
+import CreateHeaderLabel from '~/pages/Filters/Create/CreateHeaderLabel.vue'
+
+
+var modelValue = defineModel<String[]>()
```

Comment:
why is this called modelValue and not something like "keywords"?

Reply:
It doesn't change the name of the value passed to the parent. you could call it keywords in the file but I set it to modalValue so I can know that its the model getting passed up